metricbeat/docs/modules/docker/container.asciidoc

```diff
@@ -7,7 +7,6 @@ This file is generated! See scripts/mage/docs_collector.go
 
 include::../../../module/docker/container/_meta/docs.asciidoc[]
 
-This is a default metricset. If the host module is unconfigured, this metricset is enabled by default.
```

Comment:
I left these lines here, so tests can move on. But this should not be deleted. How can I fix it?

Reply:
Umm, I am trying to run metricbeat built for Linux form this branch and it seems that at least processors are not being registered, I see this:
```
Error initializing beat: error initializing processors: the processor action add_docker_metadata does not exist. Valid actions:  ...
```

The script collecting the info for the default metricsets can be run with `go run ./metricbeat/scripts/msetlists/cmd`, the output should include docker and kubernetes, it imports `github.com/elastic/beats/metricbeat/include` as metricbeat does, so it should include all modules on linux, windows and OSX.

I think there is some problem with current build tags.

Reply:
Ok, I think that `metricbeat/include/list_linux_darwin_windows.go` is not being built for linux, maybe it is only being built for `windows` platform because of this suffix.
Renaming the file to something like `metricbeat/include/list_docker.go` seems to fix the issue.